bindata/network/multus/cni-plugins-supported.yaml

```diff
@@ -0,0 +1,44 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: containernetworking-cni-plugins-supported
+  namespace: multus
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches supported CNI plugins on each node.
```

Comment:
can't we just include these containers as initContainers in the multus pod rather than launching additional daemonsets?

Reply:
done